include/substrait/function/FunctionMapping.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <memory>
+#include <unordered_map>
+
+namespace io::substrait {
+
+using FunctionMap = std::unordered_map<std::string, std::string>;
```

Comment:
I am mildly opposed to all of the `XyzPtr` using declarations (it hides the detail of whether something is `shared_ptr` or `unique_ptr`) and I am more strongly opposed to this using declaration since it would not be obvious at all what the key or value of the map are.  Constant use of these shortcuts can make a code base more difficult to newcomers.

Reply:
this class not need anymore